meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
+* TBL: with solidOs you can create a folder in your pod and put all kinds of things including a chat and that use access control systems to give groups access to them. When you look at someone's profile at the moment there is a "chat with me" button that creates a chat between the two of you.
+* WT: I don't think it is a problem to store a chatroom in a single pod, but we cannot depend on this choice of one ??? a chat based on Solid would be a perfect choice that we will encounter limits on interoperability as soon as we try to chat implementations to work together. It is not the case one follows the other in the way they store data.
+* TBL: The way we're working at the moment is that we are developing things ??? so the chat shapes, when the Oviedo students had as a student project to make a chat, they were given the chat shapes so that the requirement was that it will be compatible. So when Jackson made a chat, that is compatible I think. What the Solid CG could look at doing is making those specs match. Taking those existing shapes we have putting them into some lightweight document that other people can see. we can argue/have discussions about whether ??? and whether you prefer a different ontology.. I hope that stuff gets standardized soon. 
+* eP: I think that ??? implementation should be independent, but this task force would focus on agreeing on shapes and then have a couple of different interoperable implementations in maybe half a year and have monthly meetings to coordinate in one place.
+* TBL: If we can find people that would be great. The danger is we make a task force and nobody comes. 
+* SC: Let's start with the ACTION and get people to discuss. 
+* SC: I think people should be given an option of making a chat in their storage and send a copy to the central storage. Web Annotation has something where there is ??? I do this in dokieli. When you write an annotation, if you have a storage you can decide to put the copy of that in multiple places. If a document offers a shared ??? service and you have a storage you can use both or only one. I think the problem is not different for Solid chat but it depends on how these apps are designed. Whether to use one storage or offer people to write to their storage. Some of the problems are solved from other communities. 
 
 ### Add Solid QA
 URL: https://github.com/solid/specification/pull/495
 
 * SC: Any comments here? Objections to merge?
-
-
+* TBL: Where does this come from? 
+* SC: Part of the test suite panel work. Pete and Michiel are co-authors and we have open meetings. Also based on W3C QA work. 
+* SC: Preview of Solid QA: https://htmlpreview.github.io/?https://github.com/solid/specification/blob/53cbcefa5a93c444ad40798d01f0fa1287e4d3d8/ED/qa.html
+* SC: We're treating this as a spec in that we have some expectations from ??? so that we have a uniform approach .
+* TBL: How dow e know it won't be too bureacratic? 
```

Comment:
```suggestion
* SC: We're treating this as a spec, in that we have some expectations from ???, so that we have a uniform approach.
* TBL: How do we know it won't be too bureaucratic? 
```